packages/api/models/PortfolioDraftSummary.test.ts

```diff
@@ -0,0 +1,36 @@
+import { PortfolioDraft } from "./PortfolioDraft";
+import { portfolioDraftFields } from "./PortfolioDraftSummary";
+import { ProvisioningStatus } from "./ProvisioningStatus";
+import { exhaustivePick } from "./TypeFields";
+
+describe("Test object trimming", () => {
+  it("should not have extra attributes", async () => {
+    const portfolioDraft: PortfolioDraft = {
+      id: "test",
+      name: "test",
+      status: ProvisioningStatus.IN_PROGRESS,
+      created_at: "",
+      updated_at: "",
+      num_task_orders: 0,
+      num_environments: 0,
+      num_applications: 0,
+      num_portfolio_managers: 0,
+      funding_step: {
+        task_orders: [],
+      },
+      application_step: {
+        applications: [],
+      },
+      portfolio_step: {
+        name: "test",
+        description: "test",
+        dod_components: [],
+        portfolio_managers: [],
+      },
+    };
+    const trimmed = exhaustivePick(portfolioDraft, portfolioDraftFields);
+    expect(trimmed).not.toHaveProperty("funding_step");
+    expect(trimmed).not.toHaveProperty("portfolio_step");
+    expect(trimmed).not.toHaveProperty("application_step");
+  });
```

Comment:
Probably want to have an inverse test to make sure that it does still include at least some of the attributes that we expect to have

Reply:
Done! Also renamed the test file to make it more clear that we're not _really_ testing `PortfolioDraftSummary` itself. It's really just more of an example (and I am being lazy by not creating a specific type to use for testing).

Reply:
No worries, sometimes I create `TestEntity` classes and sometimes I get lazy, depending on how complex that is...